frontend/interfaces/team.ts

```diff
@@ -7,6 +7,8 @@ export default PropTypes.shape({
   hosts: PropTypes.number,
   members: PropTypes.number,
   role: PropTypes.string,
+  // eslint-disable-next-line react/forbid-prop-types
+  agent_options: PropTypes.string,
```

Comment:
why are we disabling  linting here? Same for line 26

Reply:
Nice catch, it was PropTypes.object, but after Zach built the API with string, changed it to string but forgot to remove disable linting 😆